pkg/resiliency/resiliency.go

```diff
@@ -63,7 +63,7 @@ type (
 		// EndpointPolicy returns the policy for a service endpoint.
 		EndpointPolicy(ctx context.Context, service string, endpoint string) Runner
 		// ActorPolicy returns the policy for an actor instance.
-		ActorPolicy(ctx context.Context, actorType string, id string) Runner
+		ActorPolicy(ctx context.Context, actorType string, id string, useRetries, useTimeout, useCB bool) Runner
```

Comment:
Is there a way to migrate this to a config struct? It will make this interface extensible and easier to read.

Reply:
We could add it into the CRD if you'd like, it just means we have to go through another naming decision. Unfortunately here, we'd also need to be pretty direct about actor mechanics in our naming. For instance we could do something like this:

```yaml
targets:
  actors:
    myActorType:
      preLock:
        retry: blah
        circuitBreaker: blah
      postLock:
        timeout: blah
```

> Also, is this "only" for tests?

No, this is needed for actors to work correctly, we just found it while testing.

Reply:
I was referring to an internal struct, not CRD.